Index: src/tools/port/conf/VERS.int_lnx
===================================================================
--- src/tools/port/conf/VERS.int_lnx	(revision 3557)
+++ src/tools/port/conf/VERS.int_lnx	(working copy)
@@ -17,7 +17,10 @@
 #set: option B64
 #set: option CAS_ENABLED
 #set: build 00
+# This should not be submitted to main
+#set: option GEO
 
+
 # Any jam definitions go here, e.g. VAR = value ;
 # Don't forget spaces around the = and before the ;
 
Index: src/tools/port/conf/VERS.a64_lnx
===================================================================
--- src/tools/port/conf/VERS.a64_lnx	(revision 3557)
+++ src/tools/port/conf/VERS.a64_lnx	(working copy)
@@ -17,7 +17,8 @@
 #set: option CAS_ENABLED
 #set: param ADI_MAX_OPERANDS 5
 #set: build 00
-#
+# This should not be submitted to main
+#set: option GEO
 
 
 # Any jam definitions go here, e.g. VAR = value ;
@@ -28,7 +29,8 @@
 
 # Hybridization...
 # a64_lnx normally builds reverse hybrid
-BUILD_ARCH = "64+32" ;	#set: arch 64+32
+# Temporarily set to pure 64
+BUILD_ARCH = "64" ;	#set: arch 64
 XERCES_ARCH ?= 64 ;
 
 PAX_CHECK = TRUE ;
Index: src/tools/port/conf/VERS.int_w32
===================================================================
--- src/tools/port/conf/VERS.int_w32	(revision 3557)
+++ src/tools/port/conf/VERS.int_w32	(working copy)
@@ -10,7 +10,10 @@
 #set: config int_w32
 #set: build 00
 #set: inc 00
+# This should not be submitted to main
+#set: option GEO
 
+
 # Any jam definitions go here, e.g. VAR = value ;
 # Don't forget spaces around the = and before the ;
 #BUILD_ICE = "TRUE" ;    #set: option BUILD_ICE
